lib/mrsk/commands/traefik.rb

```diff
@@ -54,6 +54,22 @@ def port
   end
 
   private
+    def published_ports
+      if ports = config.raw_config.dig(:traefik, "options", "publish")
+        ports.collect { |value| "--publish #{value}:#{value}" }.compact
+      else
+        ["--publish #{port}"]
+      end
+    end
+
+    def volumes
+      if volumes = config.raw_config.dig(:traefik, "options", "volumes")
+        volumes.collect { |value| "--volume #{value}" }.compact
+      else
+        ["--volume /var/run/docker.sock:/var/run/docker.sock"]
+      end
+    end
```

Comment:
I was looking to have all this be generic. That there's nothing specific about publish or volumes. That it should all just be options that turn into this. Like we do for app.